packages/core/lib/command-options.js

```diff
@@ -0,0 +1,13 @@
+const options = [
+    {
+      option: "--network <name>",
+      description:
+        "Specify the network to use. Network name must exist in the configuration."
+    },
+    {
+      option: "--config <file>",
+      description:
+        "Specify configuration file to be used. The default is truffle-config.js"
+    }
+];
```

Comment:
We discussed turning this into an object, something:

```javascript
const options = {
  network: {
    option: "--network <name>",
    description: "..."
  },
  /* ... */
}
```

Then, down below, rather than doing a filter, just use each command's specified `allowedGlobalOptions` (e.g. `allowedGlobalOptions: ["network"]`) to retrieve from that object mapping.